### DISCUSSION TITLE
新版本引入了破坏性更新？

### DISCUSSION THREAD

**Discussion by NichtsHsu** [walinejs/waline#566] [Q&A] [answered]
一直用的 `<script src="//cdn.jsdelivr.net/npm/@waline/client"></script>` 获取最新版，今天发现评论区的头像全都不显示了，回退到旧版本之后又能显示。稍微翻了一下 PR 看到了 #555，不知道与这个问题是否相关，我需要如何迁移数据？

**Comment by lizheming** [comment]
可以提供下有问题的地址，我们看一下

**Reply by NichtsHsu** [reply]
[这个](https://nihil.cc/posts/use_valine/)是指定了旧版1.0.2的，在指定之前评论区只有匿名用户有头像，指定后则正常显示

[这个](https://gm8.nihil.cc/)是默认用的最新版，现在评论区只显示匿名用户头像

**Reply by Mister-Hope** [reply]
I will also have a look later. I may import something that may break when not using latest server.

**Reply by Mister-Hope** [reply]
Confirmed as a bug existed in v1.3.6. Will be fixed in v1.3.7

**Comment by Mister-Hope** [comment, marked as answer]
Thanks for reporting ,this bug is fixed in v1.3.7.
